.travis.yml

```diff
@@ -4,6 +4,8 @@ node_js:
 script: npm test -- --coverage --runInBand
 after_success:
   - npm run codecov
+before_install:
+  - npm i -g npm@6.4.1
```

Comment:
Understand this is an interim, required npm version to get CI green for now. But do you know what error CI will give if not npm@6.4.1? (Thinking that a follow-up could fix the real underlying issue of Travis CI v.s. non npm@6.4.1)

Reply:
'twas the following:

```
> core-js@2.6.11 postinstall /home/travis/build/priceline/design-system/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"
> ejs@2.7.4 postinstall /home/travis/build/priceline/design-system/node_modules/ejs
> node ./postinstall.js
npm ERR! Cannot read property 'length' of undefined
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2019-12-24T18_33_01_894Z-debug.log
The command "eval npm ci  " failed 3 times.
The command "npm ci " failed and exited with 1 during .
```